Business Premium Update #3149 (#3426)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/teams-add-on-licensing/office-365-business-premium.md

If you have [Office 365 Business Premium](https://products.office.com/en/business/office-365-business-premium), you can use Microsoft Teams to make calls to other people in your business who are on your subscription. For example, if your business has 10 people, you can use Teams to call and IM each other.

**To make and receive calls from people external to your business, you have these options**:

> - **Option 1**. Use the [Teams app within your plan](https://products.office.com/microsoft-teams/free). If you have a very small business (for example, 1-2 people), using the Teams app is the better way to go. It's free and you are able to communicate 1:1 over chat, also group audio and video calls and share your desktop for presentations. [Compare the features, rates, and payment options](https://products.office.com/microsoft-teams/free).

Reviewer: ```suggestion
- **Option 1**. Use the [Teams app within your plan](https://products.office.com/microsoft-teams/free). If you have a very small business (for example, 1-2 people), using the Teams app is the better way to go. It's free and you are able to communicate 1:1 over chat, make group audio and video calls, and share your desktop for presentations. [Compare the features, rates, and payment options](https://products.office.com/microsoft-teams/free).
```

    
    3. Buy a [Calling Plan for Office 365](../calling-plans-for-office-365.md): you can only buy this after you buy the Phone System add-on.

>  >[!IMPORTANT]

Reviewer: ```suggestion
 > [!IMPORTANT]
```

>  > If you already have an O365 Business Premium license, when you download O365 from the portal, it already brings Teams that you can use to talk with collegues and external people, like old skype for business. You can't receive or make calls from phones at least you get two add-ins licenses:

Reviewer: ```suggestion
 > If you already have an O365 Business Premium license, when you download O365 from the portal it already includes Microsoft Teams so that you can use to talk to colleagues and external people, similar to the old Skype for Business. However, you can't receive or make calls from phones until you get at least two add-on licenses:
```
